runtime/kusama/src/lib.rs

```diff
@@ -408,8 +408,8 @@ parameter_types! {
 	// 1 hour session, 15 minutes unsigned phase, 8 offchain executions.
 	pub OffchainRepeat: BlockNumber = UnsignedPhase::get() / 8;
 
-	/// We take the top 12500 nominators as electing voters..
-	pub const MaxElectingVoters: u32 = 12_500;
+	/// We take the top 12500/22500 nominators as electing voters..
+	pub const MaxElectingVoters: u32 = prod_or_fast!(12_500, 22_500);
```

Comment:
this should either be 25k, or random_nominators_count below 22.5k

Reply:
Updated, thanks!!